src/core/Aggregators.java

```diff
@@ -141,6 +178,43 @@ public Interpolation interpolationMethod() {
     
   }
 
+  private static final class Count implements Aggregator {
```

Comment:
Also take this guy out. I'll pull in #164 since that was the first implementation.